pkg/skaffold/kubernetes/portforward/kubectl_forwarder.go

```diff
@@ -132,6 +135,9 @@ func (k *KubectlForwarder) forward(parentCtx context.Context, pfe *portForwardEn
 			//to make sure that the log monitor gets cleared up
 			cancel()
 			logrus.Debugf("port forwarding %v got terminated: %s, output: %s", pfe, err, buf.String())
+			if !strings.Contains(buf.String(), "address already in use") {
```

Comment:
create a constant locally to capture the string

Reply:
Done.